source/helpers/ApiRequest.js

```diff
@@ -18,12 +20,14 @@ const request = async (endpoint, method, data, headers) => {
   const appEnv = await StorageService.getData(APP_ENV_KEY);
   const devMode = appEnv === 'development';
   const apiKey = devMode ? env.MITTHELSINGBORG_IO_DEV_APIKEY : env.MITTHELSINGBORG_IO_APIKEY;
+  const userAgent = `${name}/${version}/${Platform.OS}/${Platform.Version}`;
 
   // Merge custom headers
   const newHeaders = {
     Authorization: token || '',
     'Content-Type': 'application/json',
     'x-api-key': apiKey,
+    'User-Agent': userAgent,
```

Comment:
Can the order of where we enter 'User-Agent' matter if the headers contain User-Agent?

Thinking if it might be better that the user-agent can not be put arguments to minimize the human factor? :)

Reply:
not sure if i understand what you mean :)

Reply:
@dannil76 You mean if `headers` already contains `User-Agent` it will override this?

Perhaps change to this then (putting `userAgent` last):

```javascript
  const newHeaders = {
    Authorization: token || '',
    'Content-Type': 'application/json',
    'x-api-key': apiKey,
    ...headers,
    'User-Agent': userAgent,
  };
```